String/CheckPalindrome.js

```diff
@@ -0,0 +1,25 @@
+// Palindrome check is case sensitive; i.e. Aba is not a palindrome
```

Comment:
please remove this commit from this pull

Reply:
I looked up on the net on how to remove commits from pull requests but I didn't get a definite solution, could you please tell me how to do this?

Reply:
you have to pull from origin/master of the original repo before pushing it to your fork copy
you can use `git remote add origin2  "link of origin repo"` and pull from it 
before pulling and making changes reset to an old commit where you want to reset
use `git reset --hard <hash value>` to get hash value use `git log`

so:
-  git log 
get hash 
-  git reset --hard  [ hash value ]
-  git remote add origin2 https://github.com/TheAlgorithms/Javascript.git
- git pull origin2 master 
and now you make changes

Reply:
see if this works

Reply:
Thank you 
I've followed all the steps but I cannot figure out what changes to make after this